rt-forecast/data/summary/cases/2020-10-11/cases_by_infection.csv

```diff
@@ -0,0 +1,147 @@
+region,date,type,median,mean,sd,lower_20,upper_20,lower_50,upper_50,lower_90,upper_90
```

Comment:
this is the internal Rt summary csv and not one for submission (that lives in submission). At the moment these are the only CrIs we ouput by default but there is a PR to make this user-settable (so that you don't have to do the read in samples and calculate quantiles step all the time).